lib/maze/schemas/trace_validator.rb

```diff
@@ -34,6 +33,10 @@ def validate
           'resourceSpans.0.scopeSpans.0.spans.0.endTimeUnixNano',
           'resourceSpans.0.scopeSpans.0.spans.0.startTimeUnixNano'
         )
+
+        unless Maze.config.client_mode_validation
```

Comment:
Other way around - it should be present for client mode
```suggestion
        if Maze.config.client_mode_validation
```